api/entity/data/policy.go

```diff
@@ -28,11 +28,11 @@ const PermissionSuper PolicyPermission = "super"
 type Policy struct {
 	Id              string             `json:"id"`
 	Name            string             `json:"name"`
-	SpiffeIdPattern string             `json:"spiffe_id_pattern"`
-	PathPattern     string             `json:"path_pattern"`
+	SpiffeIdPattern string             `json:"spiffeIdPattern"`
```

Comment:
changed the annotations to `camelCase` to be consistent with the rest of the codebase.